content/en/index.md

```diff
@@ -1,42 +1,109 @@
 ---
-title: Introduction
+title: Overview
 description: ''
 position: 1
-category: Introduction
-features:
-    - Full REST API support for non-margin accounts on listed exchanges
-    - Ticker websocket support
-    - Order book websocket support
-    - Fully multiprocessed bots with flexible arguments
-    - Quickstart access for interacting with exchanges
-    - Support for multiple portfolios on multiple exchanges, all independently
-    - Multi-process communication
-    - Long term and high resolution historical data downloads as pandas dataframes
-    - Single pip module (`pip install Blankly`)
-    - Asynchronous callbacks from ticker feeds
-    - ZeroRPC server to report to Javascript or React **
-    - Easy access to raw API calls
-    - Customizable circular buffer websocket feeds
-    - Support for Coinbase Pro & Binance sandbox modes
-    - Instant paper trade wrapper for supported exchanges
-    - Run scheduled functions natively
-    - Logs for websocket feeds
-    - Status management for purchases
-    - Create strategies using an event-based framework
-    - Instantly backtest your strategies (`strategy.backtest(to='1y')`)
-    - Automatic price downloads & caching for use in backtests
-    - Interface that allows calls to each supported exchange to be identical
-	
+category: Getting Started
 ---
 
-## We're making quantitative finance more accessible for everyone
+<blockquote> Blankly is making quantitative finance more accessible for everyone </blockquote>
+
+## The Quick Run-Down
+
+Blankly is a package that allows you to rapidly build and deploy trading models at scale in the cloud. We abstract away all the code related to connecting to exchanges (i.e. Coinbase Pro, Binance, and Stock Exchanges), order submission (Limit, Market, and Stop), and order book and price data (historical and real-time at your designated resolution) so that you can focus on building your models. 
+
+<alert type="success">
+Our goal is to take what originally takes 3-4 months of scrounging for data, connecting endpoints, and more, and make it only a few lines of code so that your workflow is optimized and easy
+</alert>
+
+
+### One Codebase, Any Exchange, Trading Multiple Entities
+
+Blankly is the only package on the market that allows you to build a model once and allow it to run on any supported exchange and trading multiple entities at a time.
+You can trade `MSFT`, `AAPL`, `PLTR` all from a single code-file with three lines of code. 
```

Comment:
3 line claim may be pushing it

Reply:
Maybe three additional lines? Was just trying to say that it's as simple as 
```python
s.add_price_event(..., 'PLTR')
s.add_price_event(..., 'MSFT')
s.add_price_event(..., 'AAPL')

Could use some rewording
```